tests/kuttl/test-ff-local/test_feature_flags.sh

```diff
@@ -59,3 +68,12 @@ if [ 'true' != "$(get_request "$CLIENT_TOKEN" "/api/client/features/$FEATURE_TOG
     echo "Feature toggle '$FEATURE_TOGGLE_NAME' should be enabled"
     exit 1
 fi
+
+# Unleash seems to refresh the cache every 5 seconds, didn't find a way to force it
+# see https://github.com/Unleash/unleash-edge/blob/12cf9e3f87099d3c0dce1884bcd305604c1e68ff/server/src/http/refresher/feature_refresher.rs#L443
+echo "Waiting for edge to sync"
+sleep 6
+
+if ! get_request_edge "$CLIENT_TOKEN" "/api/client/features/$FEATURE_TOGGLE_NAME"; then
+    echo "Feature toggle '$FEATURE_TOGGLE_NAME' should be available through edge"
```

Comment:
This would be a test failure if the toggle is not available, correct? If so, I think you need an `exit 1` within this block.

Reply:
Nice catch, I've addressed it in the place I copy pasted from too xD